src/pretix/control/logdisplay.py

```diff
@@ -198,195 +282,145 @@ def _display_order_changed(event: Event, logentry: LogEntry):
+            ), logentry, data)
+        elif data.get('first'):
+            return self.display_plain((
+                _('Position #{posid} has been checked in at {datetime} for list "{list}".'),
+                _('Position #{posid} has been checked in for list "{list}".'),
+            ), logentry, data)
+        elif data.get('forced'):
+            return self.display_plain(
+                _('A scan for position #{posid} at {datetime} for list "{list}" has been uploaded even though it has '
+                  'been scanned already.'),
+                logentry, data
             )
         else:
-            return _('Position #{posid} has been checked in for list "{list}".').format(
-                posid=data.get('positionid'),
-                list=checkin_list
-            )
-    else:
-        if data.get('forced'):
-            return _(
-                'A scan for position #{posid} at {datetime} for list "{list}" has been uploaded even though it has '
-                'been scanned already.'.format(
-                    posid=data.get('positionid'),
-                    datetime=dt_formatted,
-                    list=checkin_list
-                )
+            return self.display_plain(
+                _('Position #{posid} has been scanned and rejected because it has already been scanned before '
+                  'on list "{list}".'),
+                logentry, data
             )
-        return _(
-            'Position #{posid} has been scanned and rejected because it has already been scanned before '
-            'on list "{list}".'.format(
-                posid=data.get('positionid'),
-                list=checkin_list
-            )
-        )
-
-
-@receiver(signal=logentry_display, dispatch_uid="pretixcontrol_logentry_display")
-def pretixcontrol_logentry_display(sender: Event, logentry: LogEntry, **kwargs):
-
-    if logentry.action_type.startswith('pretix.event.order.changed'):
-        return _display_order_changed(sender, logentry)
 
-    if logentry.action_type.startswith('pretix.event.payment.provider.'):
-        return _('The settings of a payment provider have been changed.')
 
-    if logentry.action_type.startswith('pretix.event.tickets.provider.'):
-        return _('The settings of a ticket output provider have been changed.')
+class OrderConsentLogEntryType(OrderLogEntryType):
+    action_type = 'pretix.event.order.consent'
 
-    if logentry.action_type == 'pretix.event.order.consent':
+    def display(self, logentry: LogEntry, data):
         return _('The user confirmed the following message: "{}"').format(
-            bleach.clean(logentry.parsed_data.get('msg'), tags=set(), strip=True)
+            bleach.clean(data.get('msg'), tags=set(), strip=True)
         )
 
-    if logentry.action_type == 'pretix.event.order.canceled':
-        comment = logentry.parsed_data.get('comment')
+
+class OrderCancelledLogEntryType(OrderLogEntryType):
```

Comment:
```suggestion
class OrderCancelledLogEntryType(OrderLogEntryType):
```
both spellings are correct, but we use the one l everywhere

Reply:
```suggestion
class OrderCanceledLogEntryType(OrderLogEntryType):
```